data_processors/pipeline/lambdas/umccrise.py

```diff
@@ -108,11 +96,10 @@ def handler(event, context) -> dict:
     # Read input template from parameter store
     workflow_input: dict = wfl_helper.get_workflow_input()
     workflow_input['dragen_somatic_directory'] = event['dragen_somatic_directory']
-    workflow_input['fastq_list_rows_germline'] = event['fastq_list_rows_germline']
-    workflow_input['output_directory_germline'] = event['output_directory_germline']
-    workflow_input['output_directory_umccrise'] = event['output_directory_umccrise']
-    workflow_input['output_file_prefix_germline'] = event['output_file_prefix_germline']
-    workflow_input['subject_identifier_umccrise'] = subject_id
+    workflow_input['dragen_germline_directory'] = event['dragen_germline_directory']
+    workflow_input['output_directory_name'] = event['output_directory_name']
+    workflow_input['output_file_prefix'] = event['output_file_prefix']
```

Comment:
`workflow_input['output_file_prefix']` is gone? I guess, yes. Just noting. I will tackle it.

Reply:
Ah yes, output file prefix is now no longer required.